modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -45,7 +45,7 @@ When other Mule apps need access to your app's object stores, use the https://an
 
 == What are the limits on an object store?
 
-There’s no limit of the total size of an object store, but each value is limited to 10 MB. The free version also limits usage to 10 transactions per second. Premium add-on users get up to 100 TPS.
+There’s no limit of the total size of an object store, but each value is limited to 10 MB. The free version also limits usage to 10 transactions per second (TPS). Premium add-on users get up to 100 TPS per app.
```

Comment:
"There's no limit of..." 
I'm not completely sure but "limit of the..." makes some noise to me. I think this should be either limit **on**, or limit **to**. Can you please confirm with Kim?

Reply:
Yep. You're right. Thanks for your eagle eye.